ui/browser/actions/external.js

```diff
@@ -11,7 +11,9 @@ specific language governing permissions and limitations under the License.
 */
 
 import browserDB from '../../shared/browser-db';
-import { createTab, setPageDetails, duplicateTab, closeTab } from './main-actions';
+import {
+  createTab, setPageDetails, duplicateTab, closeTab, navigatePageTo,
+} from './main-actions';
```

Comment:
Was talking to @bgrins about this; I think the best solution to this is just pass in the entire props object to the children:

```
<ComponentX { ...this.props } />
<ComponentY { ...this.props } />
```

Yeah a lot of the properties will be discarded, but I don't think that'll be an issue.